### PR TITLE
Intro update

### PR DESCRIPTION
**Pull request by Jupith**

### CODE REVIEW

File: pages/intro.mdx [hunk 1]
```diff
@@ -1,48 +1,17 @@
 # Introduction
 
-You’ve found the Discord Developer Documentation! These pages are dedicated to showing you all the ways that you can use Discord to make cool stuff. Whether you’re looking to create awesome bots for your community, empower your applications with our API, or hook us right into your game with Rich Presence or the GameSDK, Discord has something for you.
+You’ve found the Unofficial Discord User Documentation! These pages are dedicated to showing you all the ways that you can use Discord to make cool stuff.
```
Comment (dolfies) [inline pages/intro.mdx:3]:
```suggestion
You’ve found the Unofficial Discord User API Documentation! These pages are dedicated to showing you all the ways that you can use Discord to make cool stuff.
```

File: pages/intro.mdx [hunk 2]
```diff
@@ -1,48 +1,17 @@
...
-- Empower them to send game invites to each other
-- Ask to Join and Spectate their friends' games
-- Spend more time playing together and less time setting up
-
-It’s free, easy, and self-serve, so check out [the GameSDK Activity Manager](/game-sdk/activities/) and get started!
-
-## Still need some help?
-
-Join the [Official Discord Developers server](https://discord.gg/discord-developers) for support and discussion regarding Discord's APIs.
+Bots and apps are not the focus of this documentation set, users are! If thats what your looking for head over to [Discord's Offical Documentation](https://discord.com/developers/docs/intro)
```
Comment (dolfies) [inline pages/intro.mdx:9]:
```suggestion
Bots and apps are not the focus of this documentation set, users are! If interactions are what you're looking for head over to [Discord's Official Documentation](https://discord.com/developers/docs/intro).
```

File: pages/intro.mdx [hunk 3]
```diff
@@ -1,48 +1,17 @@
...
+Discord offers an open API to serve requests for users, bots and OAuth2 integrations. So whether you’re making your own `!dmall` command or looking to [`Aggregate Emails`](/topics/oauth2/), we’ve got you covered.
 
-We love our developers, and we plan to keep making practical tools so that YOU can keep making cool stuff. Build a bot, integrate our account system, or put us right in your game; whatever you do, do it with Discord.
+So go do it! Go! Go [make an account](/resources/auth) and do something awesome.
 
-We can’t wait to see what you make.
+-- Random Alien
 
--- Discord
+** btw automating user accounts is "against tos" 🙄 ikr, so just a heads up doing so might get you banned **
```
Comment (dolfies) [inline pages/intro.mdx:17]:
```suggestion
**Note:** Automating user accounts is *against ToS*, 🙄 IKR, so just a heads up doing so unsafely might get you banned.
```

**Review by dolfies — CHANGES_REQUESTED** [review]

File: pages/intro.mdx [hunk 4]
```diff
@@ -1,48 +1,19 @@
...
+You’ve found the Unofficial Discord User API Documentation! These pages are dedicated to showing you all the ways that you can use Discord to make cool stuff.
 
-All of our [documentation is on GitHub](https://github.com/discord/discord-api-docs) and we <3 corrections and improvements!
+All of our [documentation is on GitHub](https://github.com/discord-userdoccers/discord-userdoccers) and we <3 corrections and improvements!
 
-## Bugs
+## ⚠️ Note
 
-If you believe you're experiencing a bug with our API or want to report incorrect documentation, open an issue on our [issue tracker](https://github.com/discord/discord-api-docs/issues).
+This documentation is not affiliated with or endorsed by Discord in any way. It is a community effort to document the API as it is used by user accounts, and is not an official source of information. Automating user accounts is _against ToS_ (🙄 IKR), so just a heads up doing so unsafely might get you banned.
```
Comment (splatterxl) [inline pages/intro.mdx:9]:
```suggestion
This documentation is not affiliated with or endorsed by Discord in any way. It is a community effort to document the API as it is used by user accounts, and is not an official source of information. Automating user accounts is against the platform Terms of Service, so just a heads up: doing so unsafely might get you banned.
```

File: pages/intro.mdx [hunk 5]
```diff
@@ -1,48 +1,19 @@
...
 
-## Bots and Apps
+## Scope
 
-Bots and apps are the lifeblood of the Discord development community. They come in all shapes and sizes, from small hobby projects for your server with friends, to huge projects that live in hundreds of thousands of servers. We love seeing the unique, fun, and sometimes downright strange (in a good way) creations that come from our community.
+Bots and interactions are not the focus of this documentation set, users are! If interactions are what you’re looking for head over to [Discord’s Official Documentation](https://discord.com/developers/docs/intro).
 
-Discord offers an open API to serve requests for both bots and OAuth2 integrations. So whether you’re making your own [`/wumpus` commands](/interactions/application-commands/) or looking to [`Log In With Discord`](/topics/oauth2/), we’ve got you covered.
+Discord offers an open API to serve requests for users, bots, and OAuth2 integrations. So whether you’re making your own `!dmall` command or looking to [`Aggregate Emails`](/topics/oauth2/), we’ve got you covered.
```
Comment (splatterxl) [inline pages/intro.mdx:15]: Do we really need this?

File: pages/intro.mdx [hunk 6]
```diff
@@ -1,48 +1,19 @@
...
-Join the [Official Discord Developers server](https://discord.gg/discord-developers) for support and discussion regarding Discord's APIs.
-
-## Go Make Cool Stuff!
-
-We love our developers, and we plan to keep making practical tools so that YOU can keep making cool stuff. Build a bot, integrate our account system, or put us right in your game; whatever you do, do it with Discord.
-
-We can’t wait to see what you make.
-
--- Discord
+-- Random Alien
```
Comment (splatterxl) [inline pages/intro.mdx:19]:
```suggestion
-- Discord User Documentation Maintainers
```

File: pages/intro.mdx [hunk 7]
```diff
@@ -1,48 +1,19 @@
...
+Bots and interactions are not the focus of this documentation set, users are! If interactions are what you’re looking for head over to [Discord’s Official Documentation](https://discord.com/developers/docs/intro).
 
-Discord offers an open API to serve requests for both bots and OAuth2 integrations. So whether you’re making your own [`/wumpus` commands](/interactions/application-commands/) or looking to [`Log In With Discord`](/topics/oauth2/), we’ve got you covered.
+Discord offers an open API to serve requests for users, bots, and OAuth2 integrations. So whether you’re making your own `!dmall` command or looking to [`Aggregate Emails`](/topics/oauth2/), we’ve got you covered.
 
-So go do it! Go! Go [make an app](https://discord.com/developers/applications) and create something awesome.
+So go do it! Go! Go [make an account](/resources/auth) and do something awesome.
```
Comment (splatterxl) [inline pages/intro.mdx:17]:
```suggestion
So go do it! Go! Go make something awesome.
```

**Review by splatterxl — CHANGES_REQUESTED** [review]

File: pages/intro.mdx (hunk 6 above)
Comment (dolfies) [reply pages/intro.mdx:19]: What are you doing with the 👽 😔

File: pages/intro.mdx (hunk 5 above)
Comment (dolfies) [reply pages/intro.mdx:15]: No but the intro page doesn't have much content

File: pages/intro.mdx (hunk 6 above)
Comment (splatterxl) [reply pages/intro.mdx:19]: it sounds unprofessional 👽

Comment (dolfies) [reply pages/intro.mdx:19]: We don't need to be that professional 👽

File: pages/intro.mdx (hunk 5 above)
Comment (splatterxl) [reply pages/intro.mdx:15]: True, maybe we could look at rewording this so as not to mention specific dodgy activities?

File: pages/intro.mdx (hunk 7 above)
Comment (dolfies) [reply pages/intro.mdx:17]: Eh I kinda like the auth link

File: pages/intro.mdx (hunk 5 above)
Comment (dolfies) [reply pages/intro.mdx:15]: Maybe 👽

**Review by dolfies — APPROVED** [review]

**Review by splatterxl — APPROVED** [review]

**Pull request merged**